bin/src/run/index.ts

```diff
@@ -19,8 +21,22 @@ export default function runRollup(command: any) {
 		}
 	}
 
-	if (command.output && command.output.dir) {
-		command.input = command._;
+	if (command.dir) {
+		if (!command._.some((input: string) => input.indexOf('=') !== -1)) {
+			command.input = command._;
```

Comment:
Maybe we should use a similar check as in the second case so that when `--input` is used, it is not overwritten with an empty array? BTW just noticed another advantage of using ' --input`: When adding input files after boolean flags, not using `--input` will confuse the command parser.

Reply:
Sure.